verl/trainer/config/ppo_trainer.yaml

```diff
@@ -1022,6 +1022,9 @@ trainer:
   # Default local directory for saving checkpoints
   default_local_dir: checkpoints/${trainer.project_name}/${trainer.experiment_name}
 
+  # Remote S3 checkpoint saving directory
+  s3_checkpoint_dir: null
+
```

Comment:
let's rename it to `default_remote_dir` and deprecate default_hdfs_dir in the future